author_api/urls.py

```diff
@@ -4,28 +4,32 @@
 urlpatterns = [
 
     # GET /login
-    url(r'^login/?$', viewsauth.Login.as_view(), name='login'),
+    url(r'^author/login/?$', viewsauth.Login.as_view(), name='login'),
 
     # POST /logout
-    url(r'^logout/?$', viewsauth.Logout.as_view(), name='logout'),
+    url(r'^author/logout/?$', viewsauth.Logout.as_view(), name='logout'),
 
     # GET POST /author/profile
-    url(r'^profile/?$', viewsauth.AuthorProfile.as_view(), name='profile'),
+    url(r'^author/profile/?$', viewsauth.AuthorProfile.as_view(), name='profile'),
 
     # GET /author/friends/:username
```

Comment:
I'm not following. This is the same url as previously. However, author used to be prefixed to all author_api urls when called from rest_api/urls.py. Now rest_api/urls.py doesn't prefix author and its prefixed within author_api as some endpoints do not need /author.